api_app/analyzers_manager/file_analyzers/filescan.py

```diff
@@ -0,0 +1,73 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import logging
+import time
+
+import requests
+
+from api_app.analyzers_manager.classes import FileAnalyzer
+from api_app.exceptions import AnalyzerRunException
+from tests.mock_utils import MockResponse, if_mock_connections, patch
+
+logger = logging.getLogger(__name__)
+
+
+class FileScan(FileAnalyzer):
+    """Filescan Class"""
+
+    def set_params(self, params):
+        self.upload_file = params.get("upload_file", True)
```

Comment:
what's the point/use-case of this parameter ?

Reply:
Sorry It was a bit redundant, removed it